src/Validator/CronExpression.php

```diff
@@ -17,15 +18,13 @@ class CronExpression extends Constraint
     public string $message = '{{ value }} is not a valid cron expression.';
 
     /**
-     * @param string|null $message
      * @param string[]|null $groups
-     * @param mixed $payload
-     * @param array $options
+     * @param mixed|null $payload
      */
     public function __construct(
         string $message = null,
         ?array $groups = null,
-        mixed $payload = null,
+        $payload = null,
```

Comment:
I am not sure about this change. Are the tests still green with thr orginal code?
mixed $payload = null,

Reply:
it broke this one:
https://github.com/Setono/CronExpressionBundle/actions/runs/11443626072/job/31836694349

but unit tests seems to be fine?

or is the problem that coding-standard vs StaticCodeAnalyis confict with each other there?

Reply:
Thanks for the link.
In my opinion, your line should be reverted.

I have created a new Issue.
https://github.com/Setono/CronExpressionBundle/issues/51

Reply:
Because our tests don't use payload, the tests still pass,
while the StaticCodeAnalyis one does fail